internal/generator/vector/conf/complex.toml

```diff
@@ -306,6 +306,7 @@ max_read_bytes = 3145728
 glob_minimum_cooldown_ms = 15000
 auto_partial_merge = true
 include_paths_glob_patterns = ["/var/log/pods/test-ns_*/*/*.log"]
+exclude_paths_glob_patterns = ["/var/log/pods/*/*/*.gz", "/var/log/pods/*/*/*.tmp", "/var/log/pods/default_*/*/*.log", "/var/log/pods/kube*_*/*/*.log", "/var/log/pods/openshift*_*/*/*.log"]
```

Comment:
@Clee2691 noted this is unnecessary given the include will never match the exclude which was a change I made for the original implementation. If it is more challenging in the implementation, I can be swayed to leave as-is

Reply:
I'd prefer to keep it there for all.   I wrote it to handle "*" but think it makes more sense to have it present.